sample-app.yaml

```diff
@@ -0,0 +1,50 @@
+# event-display app deployment
```

Comment:
can you please add the OBC defintion here as well?
would be interesting to the how the OBC is associated with the milvus definitions.
please provide an example with 2 OBCs and 2 config maps